boards/arm/nucleo_wba52cg/nucleo_wba52cg.dts

```diff
@@ -138,3 +139,32 @@ stm32_lp_tick_source: &lptim1 {
 &rng {
 	status = "okay";
 };
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 DT_SIZE_K(64)>;
+		};
+		slot0_partition: partition@10000 {
+			label = "image-0";
+			reg = <0x00010000 DT_SIZE_K(416)>;
```

Comment:
Is there a reason why 416 and not 448 like slot1? Both spaces seem to be the same size (0x70000)

Reply:
No reason. Good catch, will fix.